gm4_mending_tanks/data/mending_tanks/functions/repair.mcfunction

```diff
@@ -0,0 +1,23 @@
+#@s = experience liquid tank with item in first slot
+#run from mending_tanks:check_item
+
+
+# we will repair min(Damage, repair_points, max_repair)
+scoreboard players operation $repair gm4_lt_util = @s gm4_lt_value
+scoreboard players operation $repair gm4_lt_util += @s gm4_lt_buffer
```

Comment:
Why do you touch the buffer here?

Reply:
The fact that L8 is checking for a negative score tells me buffer can be negative. So I assume without this I managed to repair an item and extract xp points at the same time.

Relates to: #237